client/src/pages/login/index.tsx

```diff
@@ -20,19 +28,25 @@ import Spacer from '../../components/spacer/Spacer';
 import {
   checkIsExpired,
   checkIsLoggedIn,
+  clearRememberedUsername,
   clearSession,
+  getRememberedUsername,
+  setRememberedUsername,
   useAuth,
 } from '../../utils';
-import { AuthBody } from '../../types';
+import { AuthBody, UserNoAccessToken } from '../../types';
+import FallbackBackground from './assets/fallback_background.png';
```

Comment:
Will update logic for background picking once we get a nice photo of a spa. Adding this as a separate ticket for now

Reply:
Yikes content delivery gonna be a fun setup

Reply:
ticket creater here: https://github.com/KonomiAI/Chronicle/issues/251